lib/system/repr.nim

```diff
@@ -16,27 +16,27 @@ proc reprInt(x: int64): string {.compilerproc.} = return $x
 proc reprFloat(x: float): string {.compilerproc.} = return $x
 
 proc reprPointer(x: pointer): string {.compilerproc.} =
-  var buf: array[0..59, char]
-  discard c_sprintf(buf, "%p", x)
```

Comment:
But this wastes more storage. Why not pass ``addr(buf[0])`` to ``c_sprintf`` instead?